satellite/console/consoleweb/server.go

```diff
@@ -192,33 +192,32 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 
 	var projectID *uuid.UUID
 	var since, before time.Time
+	var bucketRollups []console.BucketUsageRollup
 
-	tokenCookie, err := r.Cookie("tokenKey")
+	defer func() {
+		if err != nil {
+			// TODO: use http.StatusUnauthorized status when appropriate page will be created
+			server.serveError(w, r, http.StatusNotFound)
```

Comment:
Note, use the appropriate errors codes. Using defer will make it more complicated.